master/buildbot/www/websocket.py

```diff
@@ -169,6 +169,7 @@ def parse_hybi07_frames(buf):
         if header & 0x70:
             # At least one of the reserved flags is set. Pork chop sandwiches!
             raise WSException("Reserved flag in HyBi-07 frame (%d)" % header)
+            # XXX(sa2ajj): two lines below are DEAD DEAD DEAD
```

Comment:
Yes; but note that this code is from Twisted.  It may not be used at all, or it may be possible to use a committed version.